### PR TITLE
C117607: Adding hard break line and closing square bracket

### PR DESCRIPTION
**Pull request by v-maudel**
Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  
Description: Missing hard break line and closing square bracket are creating the issue

### CODE REVIEW

**Comment by v-maudel** [comment]
Hi @rpetrusha, can you please help by merging this PR?
Thanks in advance

File: docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ A function is considered dynamic if it has no metadata. Certain methods like IL
 
 **Library:** CorGuids.lib
 
-**.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)
+**.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)]
```
Comment (mairaw) [inline docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md]: I don't think we need the extra spaces here

```suggestion
**.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)]
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @v-maudel for fixing the include syntax but I don't think we need to add the spaces.

File: docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md (hunk 1 above)
Comment (v-maudel) [reply docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md]: Thanks @mairaw , both changes were asked by the bugfixing team but we can try the fix without the hard breakline

**Comment by mairaw** [comment]
Also icorprofilerinfo8-getfunctionfromip3-method.md has the same problem. Would you like to fix that in this PR?

**Comment by v-maudel** [comment]
Hi @rpetrusha, can you please help by merging this PR?
Thanks in advance

> 
> 
> Also icorprofilerinfo8-getfunctionfromip3-method.md has the same problem. Would you like to fix that in this PR?

I just was on it :)
Done here: https://github.com/dotnet/docs/pull/14318

**Review by mairaw — APPROVED** [review]
LGTM

**Comment by v-maudel** [comment]
Thanks @mairaw

**Pull request merged**